include/curl/curl.h

```diff
@@ -685,8 +685,10 @@ typedef enum {
 typedef CURLcode (*curl_conv_callback)(char *buffer, size_t length);
 
 typedef CURLcode (*curl_ssl_ctx_callback)(CURL *curl,    /* easy handle */
-                                          void *ssl_ctx, /* actually an
-                                                            OpenSSL SSL_CTX */
+                                          void *ssl_ctx, /* actually an OpenSSL
+                                                          or WolfSSL SSL_CTX,
```

Comment:
Is there space here wihint our 80 character limit to align the first character of each addition comment line with the start to `actually`?

Reply:
For these lines, yes, but the "mbedtls_ssl_config */" line would be too long then. I'll have just that one be "wrong"